admin_guide/manage_users.adoc

```diff
@@ -66,6 +105,47 @@ authentication provider after starting to use {product-title}, the user names
 that overlap will not work because of the entries in the identity list, which
 will still point to the old authentication method.
 
+[[admin-guide-managing-users-creating-groups]]
+== Creating Groups
+
+While a user is an entity making requests to {product-title}, users can be
+organized into one or more groups made up from a set of users. Groups are useful
+for managing many users at one time, such as for authorization policies, or to
+grant permissions to multiple users at once.
+
+If your organization is using LDAP, you can synchronize any LDAP records to
+{product-title} so that you can configure groups on one place. This presumes
+that information about your users is in an MDAP server. See the
+xref:../install_config/syncing_groups_with_ldap.adoc#install-config-syncing-groups-with-ldap[Synching
+groups with LDAP section] for more information. If you are not using LDAP, you
+can use the following procedure to manually create groups.
+
+To create a new group:
+
+----
+# oc adm groups new <group_name> <user1> <user2>
```

Comment:
@enj @openshift/sig-security this stands out as a command that should be made into `oc create group`

Reply:
> @enj @openshift/sig-security this stands out as a command that should be made into oc create group

the doc is fine.